workshop/workshop-structure.md

```diff
@@ -47,7 +50,7 @@ You can read more about how we use Zoom and Slack in the [Zoom procedures](../vi
 | I need something clarified before we move on with instruction | [Use Zoom chat to message the meeting host with your question or use the `Raise Hand` button on Zoom](../virtual-setup/zoom-procedures.md#interacting-with-instructors-and-providing-feedback) |
 | I am stuck with an error message and am no longer able to follow the hands-on instruction | [Use Zoom chat](../virtual-setup/zoom-procedures.md#chat) to [message the meeting host and let them know you need 1:1 assistance](../virtual-setup/zoom-procedures.md#asking-for-11-help-during-instruction) - they will pair you with an available instruction in a [Zoom breakout room](../virtual-setup/zoom-procedures.md#joining-a-breakout-room) |
 | I have a general or conceptual question that can be addressed later in the lecture or asynchronously | Post in the [training-specific Slack channel](../virtual-setup/slack-procedures.md#using-the-training-specific-channel) |
-| I am in a breakout room and want to ask my instructor a question about the exercise | You can [unmute yourself](../virtual-setup/zoom-procedures.md#audio) and ask or [use the chat window to message the instructor](../virtual-setup/zoom-procedures.md#chat) who will be designated as a Host or Co-host of the meeting; you may be asked to [share your screen](../virtual-setup/zoom-procedures.md#sharing-your-screen-in-zoom) |
+| I am in a breakout room and want to ask my instructor a question | [Use the `Raise Hand` button on Zoom](../virtual-setup/zoom-procedures.md#interacting-with-instructors-and-providing-feedback), your instructor will ask you to [unmute yourself](../virtual-setup/zoom-procedures.md#audio) and ask your question. You can [use the chat window to message the instructor](../virtual-setup/zoom-procedures.md#chat), who will be designated as a Host or Co-host of the meeting, if you prefer. You may be asked to [share your screen](../virtual-setup/zoom-procedures.md#sharing-your-screen-in-zoom) if you encounter errors. |
```

Comment:
I'm of two minds about raising hands in small groups; seems reasonable to encourage politeness but I worry about added friction.

Reply:
Same, but the thought process here was avoiding 3 people starting talking at once. It could be up to the instructor's discretion to say okay don't worry about raising your hand if it seems like this step is a barrier.